backend/src/apiserver/resource/resource_manager.go

```diff
@@ -519,7 +519,7 @@ func (r *ResourceManager) ListJobs(filterContext *common.FilterContext,
 func TerminateWorkflow(wfClient workflowclient.PipelineRunInterface, name string) error {
 	patchObj := map[string]interface{}{
 		"spec": map[string]interface{}{
-			"status": "Cancelled",
+			"status": common.GetTerminateStatus(),
```

Comment:
This is the default behavior, do we support user to overwrite it by explicitly specifying the parameter?

Reply:
This PR allows the admin to configure this status, not users. If we want to allow users to configure this, we need to have some new annotations/dsl to indicate this new status.